src/App.js

```diff
@@ -1,18 +1,48 @@
 import React, { Component } from 'react'
-import logo from './logo.svg'
+import {
+  FormGroup,
+  FormControl
+} from 'react-bootstrap'
 import './App.css'
+import PasteInlineForm from './pasteInlineForm.js'
 
 class App extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      codeText: '',
+      codeTitle: '',
+      codeAuthor: ''
+    }
+
+    this.handleTextChange = this.handleTextChange.bind(this)
+  }
+
+  handleTextChange (e, formName) {
+    this.setState({[formName]: e.target.value})
```

Comment:
You should have a guard in this code to prevent formName from being something other than `codeText` , `codeTitle`, or `codeAuthor`